schemas/location.py

```diff
@@ -50,10 +53,10 @@ class CreateLocation(BaseCreateModel, ValidateLocation):
     notes: List[CreateNote] = []
     point: str  # point is required and should be in WKT format
     elevation: float
-    elevation_accuracy: float | None = None
-    elevation_method: ElevationMethod | None = None
-    coordinate_accuracy: float | None = None
-    coordinate_method: CoordinateMethod | None = None
+    # elevation_accuracy: float | None = None
```

Comment:
Why not delete these fields if they're not going to be used?

Reply:
I decided to comment them out for now because we may want to include them in the future, so they're here for notes. I can remove those comments if it's bad style to keep these in.